microprofile-lra/README.adoc

```diff
@@ -680,6 +680,23 @@ You should see a message in the server log indicating that the archive deployed
 
 :leveloffset!:
 
+== Access the quickstart application
+
+You can either access the application via your browser at http://localhost:8080/microprofile-lra/. You can also access it from the command line:
+
+[source,bash]
+----
+$ curl http://localhost:8080/microprofile-lra/
+----
+
+It should return a simple output:
+
+[source]
+-----
+Hello! Microprofile-Lra Quickstart deployed successfully! You can find the available operations in the included README file.
```

Comment:
I think this should be `MicroProfile LRA`

Reply:
Let's revert and not do any README changes wrt adding a root deployment page, please. As I said in the jaxrs-client PR we should later workout a common strategy for documenting this in a /shared-docs included section.

Reply:
Reverted the readme changes.